app/controllers/markets_controller.rb

```diff
@@ -1,7 +1,22 @@
 class MarketsController < ApplicationController
+  def search
+  end
+
   def index
+    if params[:use_my_location] != "0"
```

Comment:
I believe lines 6-8 here will need to change when we learn how to use browser api location.